web/templates/popovers/left_sidebar/left_sidebar_topic_actions_popover.hbs

```diff
@@ -38,7 +38,7 @@
         {{else}}
             {{!-- Group 2 --}}
             <li role="separator" class="popover-menu-separator"></li>
-            {{#if (and is_development_environment (or is_moderator is_realm_admin))}}
+            {{#if can_summarize_topics}}
```

Comment:
I think we do not need `is_development_environment` check anymore since we can keep this disable in production by using server setting which is used to get the value of `can_summarize_topics`.

Reply:
Yes, exactly, being able to replace the hacky `is_development_environment` checks is one of the nice benefits of adding these settings now.